src/inmanta/data/__init__.py

```diff
@@ -4463,6 +4463,9 @@ async def get(
 
     @classmethod
     def new(cls, environment: uuid.UUID, resource_version_id: m.ResourceVersionIdStr, **kwargs: Any) -> "Resource":
+        if "attributes" in kwargs and "requires" in kwargs["attributes"]:
```

Comment:
This I don't inderstand?

Reply:
It's a guard to make sure that the `requires` field in the attributes is always a rid and not and rvid.